packages/x-date-pickers/src/TimeClock/Clock.tsx

```diff
@@ -332,6 +332,14 @@ export function Clock<TDate extends PickerValidDate>(inProps: ClockProps<TDate>)
         handleValueChange(viewValue - keyboardControlStep, 'partial');
         event.preventDefault();
         break;
+      case 'PageUp':
+        handleValueChange(viewValue + 5, 'partial');
```

Comment:
Nitpick: WDYT if we would round the the to the nearest `% 5 === 0`? 🤔 
Given the layout of the minutes, it seems that this could be a nice UX. 🤷 

https://github.com/user-attachments/assets/bfd2ca9d-6d89-4834-bfa6-9a64e7a03b57

Reply:
> Don't you think we should keep the consistency among the components that are used combined ?

Fair points. 👍 
Let's stick with consistency, at least for now. 👍 
We can re-evaluate when the `timeSteps` prop impacts fields and views alike. 👌